ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderTest.java

```diff
@@ -645,26 +645,31 @@ private void givenWindowedSourceTable() {
   private void givenUnwindowedSourceTable() {
     when(buildContext.buildKeySerde(any(), any(), any())).thenReturn(keySerde);
     givenConsumed(consumed, keySerde);
+    Formats genericFormat = Formats.of(keyFormatInfo, valueFormatInfo, KEY_FEATURES, VALUE_FEATURES);
```

Comment:
How about just `formats` instead of `genericFormat`? (Unless the "generic" bit adds anything to the name, shorter variable names are preferred.)

Since this is always created in the same way, we can create it in the `setUp()` method instead of in multiple places throughout the tests. (See how `planBuilder` is being created as an example.)